packages/rdma-core/rdma-core.spec

```diff
@@ -0,0 +1,165 @@
+Name: %{_cross_os}rdma-core
+Version: 53.1
+Release: 1%{?dist}
+Summary: RDMA core userspace libraries and daemons
+License: MIT OR GPL-2.0-only
+Source0: https://github.com/linux-rdma/rdma-core/releases/download/v%{version}/rdma-core-%{version}.tar.gz
+Source100: rdma-core-tmpfiles.conf
+
+BuildRequires: cmake >= 2.8.11
+BuildRequires: %{_cross_os}libnl-devel
+BuildRequires: %{_cross_os}glibc-devel
+Requires: %{_cross_os}libnl
+
+%description
+%{summary}.
+
+%package devel
+Summary: RDMA core development libraries and headers
+# For now we are only providing the libibverbs-devel as it
+# will be needed to build EFA Device Plugin.
+Provides: %{_cross_os}libibverbs-devel
+
+%description devel
+%{summary}.
+
+%prep
+%autosetup -n rdma-core-%{version} -p1
+
+%build
+%{cross_cmake} . \
+  -DNO_PYVERBS=1 \
+  -DNO_MAN_PAGES=1 \
+  -DENABLE_STATIC=1 \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DCMAKE_INSTALL_PREFIX:PATH=%{_cross_prefix} \
+  -DCMAKE_INSTALL_LIBDIR:PATH=%{_cross_libdir} \
+  -DCMAKE_INSTALL_BINDIR:PATH=%{_cross_bindir} \
+  -DCMAKE_INSTALL_SBINDIR:PATH=%{_cross_sbindir} \
+  -DCMAKE_INSTALL_INCLUDEDIR:PATH=%{_cross_includedir} \
+  -DCMAKE_INSTALL_SYSCONFDIR:PATH=%{_cross_sysconfdir} \
+  -DCMAKE_INSTALL_FULL_SYSCONFDIR:PATH=%{_cross_sysconfdir} \
+
+%make_build
+
+%install
+%make_install
+
+install -d %{buildroot}%{_cross_tmpfilesdir}
+install -p -m 0644 %{S:100} %{buildroot}%{_cross_tmpfilesdir}/rdma-core.conf
+
+install -d %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}/libibverbs.d
+install -p %{buildroot}%{_cross_sysconfdir}/libibverbs.d/efa.driver %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}/libibverbs.d
```

Comment:
Is `efa.driver` a configuration file?

Reply:
The `*.drivers` are device-specific plug-in [ibverbs userspace drivers](https://github.com/linux-rdma/rdma-core/blob/master/redhat/rdma-core.spec#L189-L204). And efa.driver corresponds to the "Amazon Elastic Fabric Adapter" provider. Other driver information:
```
bnxt_re.driver       - Broadcom NetXtreme-E RoCE
cxgb4.driver        - Chelsio T4/T5/T6 iWARP/RoCE
erdma.driver        - Elastic RDMA driver
hfi1verbs.driver    - Intel Omni-Path HFI
hns.driver          - HiSilicon Network Subsystem
ipathverbs.driver   - QLogic InfiniBand
irdma.driver        - Intel RDMA
mana.driver         - Microsoft Azure Network Adapter
mlx4.driver         - Mellanox/NVIDIA ConnectX-3
mlx5.driver         - Mellanox/NVIDIA ConnectX-4/5/6
mthca.driver        - Mellanox HCA
ocrdma.driver       - Emulex OneConnect RoCE
qedr.driver         - QLogic FastLinQ RoCE
rxe.driver          - Soft RoCE
siw.driver          - Software iWARP
vmw_pvrdma.driver   - VMware Paravirtual RDMA
```